go/libkb/pgp_key.go

```diff
@@ -255,6 +255,10 @@ func PGPKeyRawToArmored(raw []byte, priv bool) (ret string, err error) {
 	return
 }
 
+func (k *PGPKeyBundle) SerializePrivate(w io.Writer) error {
+	return k.Entity.SerializePrivate(w, &packet.Config{ReuseSignaturesOnSerialize: !k.Generated})
```

Comment:
If `ReuseSignaturesOnSerialize` is false, remind me what happens? Do signatures just get dropped, or do new signatures also get made?